src/public/meeting.ts

```diff
@@ -143,4 +154,73 @@ export namespace meeting {
     const messageId = sendMessageRequestToParent('meeting.getAuthenticationTokenForAnonymousUser');
     GlobalVars.callbacks[messageId] = callback;
   }
+
+  /**
+   * Allows an app to get the state of the live stream in the current meeting
+   * @param callback Callback contains 2 parameters: error and liveStreamState.
+   * error can either contain an error of type SdkError, in case of an error, or null when get is successful
+   * liveStreamState can either contain a LiveStreamState value, or null when operation fails
+   */
+  export function getLiveStreamState(
+    callback: (error: SdkError | null, liveStreamState: LiveStreamState | null) => void,
+  ): void {
+    if (!callback) {
+      throw new Error('[get live stream state] Callback cannot be null');
+    }
+    ensureInitialized();
```

Comment:
Please pass FrameContext.sidePanel in all APIs to endure these are only available in sidePanel.

Reply:
fixed